modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -28,7 +28,7 @@ spec:
     deviceID: "<device_id>" <10>
     pfNames: ["<pf_name>", ...] <11>
     rootDevices: ["<pci_bus_id>", "..."] <12>
-	netFilter: "<filter_string>" <13>
+  netFilter: "<filter_string>" <13>
```

Comment:
Looks like the indent is not perfect yet, we need to keep it the same as other selectors like rootDevices.

Reply:
Indent still doesn't look correct?